app/controllers/comfy/admin/cms/seos_controller.rb

```diff
@@ -1,4 +1,10 @@
 class Comfy::Admin::Cms::SeosController < Comfy::Admin::Cms::BaseController
   def index
   end
+
+  def check
+    webpage = 'http://www.ita-online.info'
```

Comment:
I would make this an input field in a form, or use the root url of the app instead of hard coding it. I would consult Adam or the SEO contact we have.